aehmc/window_adaptation.py

```diff
@@ -54,12 +62,16 @@ def one_step(
         warmup_state, parameters = update_adapt(
             warmup_step, warmup_state, parameters, chain_state
         )
-
+        da_state = warmup_state[0]
         return (
             chain_state[0],  # q
             chain_state[1],  # potential_energy
             chain_state[2],  # potential_energy_grad
-            *warmup_state[0],
+            da_state.step,
+            da_state.iterates,  # log_step_size
+            da_state.iterates_avg,  # log_step_size_avg
+            da_state.gradient_avg,
+            da_state.shrinkage_pts,  # mu
```

Comment:
What are these comments indicating?

Reply:
The aual averaging state's "iterates" is represented by the `log_step_size` variable and the "shrinkage_pts" is represented by the `mu` variable in the context of this function. Is this correct, @rlouf ?